frontend/components/AddHostsModal/PlatformWrapper/PlatformWrapper.tsx

```diff
@@ -536,13 +536,13 @@ const PlatformWrapper = ({
           >
             <>
               Include&nbsp;
-              <TooltipWrapper
+              <NewTooltipWrapper
                 tipContent={
-                  "Include Fleet Desktop if your’re adding workstations."
+                  "Include Fleet Desktop if your're adding workstations."
```

Comment:
Typo fix (you're not your're)

Reply:
Do we want to use `&apos;` consistently for apostrophes?

Reply:
`NewTooltipWrapper.tipContent` has the type `string | JSX.Element | undefined` – if passing in a `JSX.Element`, named character references like `&apos;` are necessary. If passing in a `string`, they're not.